source/Server.Extensibility/Server.Extensibility.csproj

```diff
@@ -26,6 +26,8 @@
     <None Remove="Extensions\Infrastructure\Web\Content\**" />
   </ItemGroup>
   <ItemGroup>
+    <PackageReference Include="Autofac" Version="4.9.4" />
```

Comment:
I am reverting the version of Autofac to what we currently have in server. This was previously bumped to `6.0.0` from `4.9.4`, but has not been included in server so feels super risky to do.